GameServer.py

```diff
@@ -6,42 +6,50 @@
 from PodSixNet.Server import Server
 
 from ClientHandler import ClientHandler
-
+from PlayerInfo import PlayerInfo
 
 class GameServer(Server):
     channelClass = ClientHandler
+    idArray = [0, 0, 0]
+    playersInfoArray = []
 
     def __init__(self, *args, **kwargs):
         Server.__init__(self, *args, **kwargs)
         self.players = WeakKeyDictionary()
         print('Server launched')
 
-    def AddPlayer(self, player):
-        self.players[player] = True
-        print("New Player: " + str(player.addr))
-        print("Connected players: ", [p for p in self.players])
-
-    def DelPlayer(self, player):
-        print("Deleting Player" + str(player.addr))
+    def addPlayer(self, player):
+        playerinfo = PlayerInfo(0,0, self.getNewId())
+        self.players[player] = playerinfo
+        self.playersInfoArray.append({"id": playerinfo.id, "x": playerinfo.x, "y": playerinfo.y})
```

Comment:
So do we stick with array of dicts or WeakKeyDictionary?

Reply:
I delete WeakKeyDictionary and leave only playersInfoArray